lib/logger/index.ts

```diff
@@ -1,15 +1,15 @@
 import is from '@sindresorhus/is';
 import * as bunyan from 'bunyan';
-import * as shortid from 'shortid';
+import { nanoid } from 'nanoid';
```

Comment:
What's the benefit?

Reply:
Shortid is deprecated, and its typings seemed to be missing

Reply:
Ok, i saw we already have nanoid as transitive dep.